osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/monitoring/AbstractGetPowerQualityProfileHandler.java

```diff
@@ -6,7 +6,7 @@
  *
```

Comment:
most code in this class is the old executor, I rewrote some, and some code I kept intact.